ext/wry_ruby/src/application/event_loop.rs

```diff
@@ -0,0 +1,39 @@
+use magnus::{
+    function, Error, Object,
+};
+
+use std::marker::PhantomData;
+use std::sync::atomic::{AtomicPtr};
+
+use wry::application::event_loop::{
+      EventLoop as EventLoopImpl, 
+};
+
+#[magnus::wrap(class = "EventLoop")]
+pub struct EventLoop {
+    inner: SafeWrapper<EventLoopImpl<()>>,
+}
+
+pub struct SafeWrapper<T> {
+    event_loop: AtomicPtr<T>,
+    _marker: PhantomData<*mut T>,
+}
+
+unsafe impl<T> Send for SafeWrapper<T> {} 
+
+impl EventLoop {
```

Comment:
i am not implementing event_loop further(as for now) as per discussion in discord